website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -64,7 +64,18 @@ If you’re using any undocumented and unsupported API endpoints, please note th
 Are you using a user API key today to access <Constant name="cloud" /> APIs in any of your workflows? If not, you don’t have any action to take. If you are using a user API key, please follow the instructions below. 
 
 1. Make a list of all the places where you’re making a call to the <Constant name="cloud" /> API using the <Constant name="cloud" /> user API key. 
-2. Create a new personal access token under **Account Settings → API Tokens → Personal Tokens.** 
-3. Create and copy the new PAT to replace the old user API key. 
+2. Create a new PAT under **Account Settings → API Tokens → Personal Tokens.** For instructions, see [Create a personal access token](#create-a-personal-access-token).
+3. Replace the API key in your APIs with the PAT you created. You can use a PAT wherever you previously used an API key.
+    
+    To replace the API key with a PAT, include the PAT in the Authorization header of your API requests. For example: `Authorization: Bearer <your-token>`.
+
+    Make sure to replace `<your-token>` with the new PAT you created.
+
+    :::note
+
+    You do not need to replace your API key with a PAT in <Constant name="cloud" />. The option to rotate API keys is used for rotating existing API keys, not for replacing them with PATs.
```

Comment:
Adding some extra language and leading with the API key rotation might help us be extra clear
```suggestion
     The option to rotate API keys is used for existing API keys, not for replacing them with PATs. You do not need to replace your API key with a PAT in the <Constant name="cloud" /> UI.
```